src/android/SimpleCameraPreview.java

```diff
@@ -347,7 +347,8 @@ public void onProviderDisabled(String provider) {
 
     private boolean capture(boolean useFlash, CallbackContext callbackContext) {
         if (fragment == null) {
-            callbackContext.error("Camera is closed");
+            PluginResult pluginResult = new PluginResult(PluginResult.Status.OK, false);
```

Comment:
in case of capture, should we return Status.OK, since for success callback the js side is exepecting a nativePath, do we need to return error callback in this context.